FABulous/FABulous_CLI/helper.py

```diff
@@ -216,7 +216,7 @@ def _copy_template_safely(template_ref: Traversable, target_dir: Path) -> None:
     set_key(
         env_file,
         "FAB_MODELS_PACK",
-        str(project_dir.absolute() / "Fabric" / f"models_pack.{new_suffix}"),
+        str(f"{project_dir.name}/Fabric/models_pack.{new_suffix}"),
```

Comment:
```suggestion
        str(Path(project_dir.name) / "Fabric" / f"models_pack.{new_suffix}"),
```
Maybe this is better? Since this will auto adapt to the systems seperator?